about.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>About Drummers Summit</title>
+</head>
+<body>
+    <header>
+        <ul class="first-navbar desktop">
```

Comment:
- [ ] Kindly follow the given design by removing the logo for the mobile version of the site

Your design
![image](https://user-images.githubusercontent.com/81932478/196777140-5b8b0f5b-66b9-4ac4-8445-1b91b2fcc20b.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/196777259-388b8f20-94fe-478c-b640-dcec8e3924fa.png)